lawnchair/res/values/strings.xml

```diff
@@ -343,6 +343,8 @@
 
     <string name="pref_all_apps_remember_position_title">Remember Position</string>
     <string name="pref_all_apps_remember_position_description">Remember the position when you left off instead of the top</string>
+    <string name="pref_all_apps_show_scrollbar_title">Show Scrollbar</string>
+    <string name="pref_all_apps_show_scrollbar_description">Show the scrollbar in the app drawer</string>
```

Comment:
We won't need to have a description for this since the label is already self-explanatory.

If you insist on a description:
```suggestion
    <string name="pref_all_apps_show_scrollbar_description">To quickly navigate apps alphabetically</string>
```